source/style-guide/seo-guidelines.txt

```diff
@@ -0,0 +1,84 @@
+.. _seo-guidelines:
+
+=====================================
+Search Engine Optimization Guidelines
+=====================================
+
+.. default-domain:: mongodb
+
+Search Engine Optimization (SEO) considers the actual terms that users 
+enter into serach engines (keywords) and employs best practices to 
+improve traffic to web pages from search engines. The following 
+guidance should help you optimize your documentation pages for search.
+
+Titles
+------
+
+Title pages and subsections according to the following SEO best 
+practices:
+
+- Use a maximum of 70 characters.
+
+- Include target keywords.
+
+- Avoid excessive or irrelevant words (keyword stuffing).
+
+- Use a unique title for every page.
```

Comment:
Questions: 
Would this guidance apply to pages that might share titles in different sets of documentation? For example, "find a document" could be a title of a page in Server, Drivers, mongosh, etc. 
Also, if this applies to subsections, then should we avoid using section titles like "Example", "Syntax", and "Further Learning"?

Reply:
This mostly concerns titles at the page level even between doc sets. Identical titles might compete in search results.